src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons in the address book by alphabetical order temporarily.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        
+        List<Person> allPersons = addressBook.getAllPersons().mutableListView();
+
+        Collections.sort(allPersons, new Comparator() {

```

Comment:
Is there a better way to implement the sorting in an OOP fashion? This is more like FP.

Reply:
Yep I could have let Person extend the Comparable class instead! Thanks for pointing this out :)